Frameworks/CoreText/CTLine.mm

```diff
@@ -289,20 +289,19 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
 
 /**
  @Status Interoperable
- @Notes
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
     _CTLine* line = static_cast<_CTLine*>(lineRef);
 
-    if (!line || [line->_runs count] == 0) {
-        return 0;
-    }
+    RETURN_RESULT_IF((!line || CFArrayGetCount(line->_runs) == 0), 0);
```

Comment:
nit: parenthesize the two sides of || rather than, or in addition to, both in one
as is, my brain keeps wanting to read it as
( (!line || CFArrayGetCount(line->_runs)) == 0 )